update Marathi translations and add missing keys

the-carnage opened the pull request:
This PR updates the Marathi localization file (src/i18n/locales/mr.js) to include missing translations for keys such as open_recent, no_saved_diagrams, and others that were previously defaulting to English or missing. This ensures a more complete and consistent Marathi user experience.
before:- 
<img width="1496" height="544" alt="Screenshot 2025-11-23 at 10 58 05 PM" src="https://github.com/user-attachments/assets/8819b8a6-5c56-4cd1-9515-fa7f65c5be4f" />
after:-
<img width="1494" height="609" alt="Screenshot 2025-11-23 at 10 58 28 PM" src="https://github.com/user-attachments/assets/79e60944-20f8-4050-8dbf-3699652a7a96" />

1ilit reviewed (changes requested):
please remove the changes in package-lock.json

the-carnage commented:
> please remove the changes in package-lock.json

I’ve removed all the changes from package-lock.json and updated the PR accordingly. Please let me know if anything else needs to be adjusted.

1ilit reviewed (approved):
Thank you
